tests/test_cases/issue_1279/issue_1279.py

```diff
@@ -9,16 +9,15 @@
     expect_error=cocotb.result.SimFailure,
     stage=1,
 )
-def test_sim_failure_a(dut):
+async def test_sim_failure_a(dut):
     # invoke a deadlock, as nothing is driving this clock
-    yield cocotb.triggers.RisingEdge(dut.clk)
+    await cocotb.triggers.RisingEdge(dut.clk)
 
 
 @cocotb.test(
     skip=cocotb.SIM_NAME.lower().startswith("riviera"),  # gh-1859
     expect_error=cocotb.result.SimFailure,
     stage=2,
 )
-def test_sim_failure_b(dut):
-    yield cocotb.triggers.NullTrigger()
-    raise cocotb.result.TestFailure("This test should never run")
+async def test_sim_failure_b(dut):
+    assert False, "This test should never run"
```

Comment:
Are we sure the null trigger was not an important part of this test?

Reply:
#1279 is about the next test running at all. So I don't think it was.